sqlglot/dialects/starrocks.py

```diff
@@ -54,6 +163,7 @@ class Generator(MySQL.Generator):
         EXCEPT_INTERSECT_SUPPORT_ALL_CLAUSE = False
         JSON_TYPE_REQUIRED_FOR_EXTRACTION = False
         PARSE_JSON_NAME: t.Optional[str] = "PARSE_JSON"
+        WITH_PROPERTIES_PREFIX = "PROPERTIES"
```

Comment:
Just making sure - starrocks doesn't have a `WITH (k1=v1, k2=v2, ...)` DDL property syntax?

Reply:
Yes, in the documentation it provides the pattern for create.
https://docs.starrocks.io/docs/sql-reference/sql-statements/data-definition/CREATE_TABLE/#syntax
There is no other pattern provided in `Parameters` - `properties` either.

When using `with` to execute in StarRocks, it will get the error:
![image](https://github.com/user-attachments/assets/fa162f67-2084-4d3e-a186-28ac74625f67)


```
Caused by: java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'with' at line 9
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:767)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:652)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more